viewer/viewer-awt/src/main/java/org/icepdf/ri/common/utility/search/SearchTextTask.java

```diff
@@ -346,7 +348,7 @@ public boolean isDestinations() {
         private final String pattern;
 
         // parent search panel
-        private SearchPanel searchPanel;
+        private BaseSearchModel searchPanel;
```

Comment:
Sorry, one more rename request

Reply:
No problem, it's better to be consistent and have clean code rather than rush things. I renamed it to `searchModel` (it seemed better than `baseSearchModel` IMO, given that having a `baseSearchModel` would imply the existence of a `searchModel`)

Reply:
Works for me,  thanks!